### DISCUSSION TITLE
Welcome to flutter_commun_app Discussions!

### DISCUSSION THREAD

**Discussion by TheAlphamerc** [TheAlphamerc/flutter_commun_app#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by imsuriyan** [comment]
How many days it will take to fiNish it?
Eagerly waiting 😊

**Reply by TheAlphamerc** [reply]
I think it can take upto 3-4 months to complete the basic functionality. This is just a prediction, in real the time can be more or less.
